Meadowlark-js/backends/meadowlark-aws-dynamodb-opensearch/src/BaseDynamoRepository.ts

```diff
@@ -229,13 +230,13 @@ export function constructAssignableDeleteItem(
   resourceInfo: ResourceInfo,
   documentInfo: DocumentInfo,
 ): TransactWriteItem | null {
-  if (documentInfo.assignableInfo == null) return null;
+  if (documentInfo.superclassInfo == null) return null;
 
-  // TODO: Note for the future, this assumes the "assignable to" entity is in the same project/namespace as the given entity
+  // TODO: Note for the future, this assumes the "superclass" entity is in the same project/namespace as the given entity
```

Comment:
I just started https://techdocs.ed-fi.org/display/EFTD/Meadowlark+-+Miscellaneous+Notes to help remember issues like this. I'm not ready to fill up the Jira backlog just yet. Don't want to get too far ahead of ourselves.